xla/service/cpu/onednn_ops_rewriter.cc

```diff
@@ -84,6 +86,9 @@ auto MaxReduce(HloInstruction** instr) {
       m::Reduce(instr).WithPredicate(is_valid_reduce_max).WithOneUse());
 }
 
+// Match the softmax patter with divide instruction as root node.
+// Here we pass 'instr' as root node and return the producer HloInstruction
```

Comment:
Please make sure to format sentences correctly, e.g., here:

"return the producer HloInstruction[. The a]xis on which softmax is applied is stored in 'axis'".

Reply:
Done